tests/templates/test_subroutines/test_aqft.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2018-2023 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for the aqft template.
+"""
+import pytest
+
+import numpy as np
+import pennylane as qml
+
+
+class TestAQFT:
```

Comment:
Add a test verifying the decomposition is what is expected for a given set of wires, this could also be an integration test which tests if apply AQFT to a known input state gives you the expected output state 👍🏼